### PR TITLE
Rename default config backup under /opt/

### PR DESCRIPTION
**Pull request by daipom**

### CODE REVIEW

File: fluent-package/Rakefile [hunk 1]
```diff
@@ -478,7 +478,7 @@ class BuildTask
       desc "Create fluent-package configuration files from template"
       task :fluent_package_config do
         fluentd_conf = "etc/#{PACKAGE_DIR}/#{SERVICE_NAME}.conf"
-        fluentd_conf_default = "opt/#{PACKAGE_DIR}/share/#{COMPAT_SERVICE_NAME}.conf.tmpl"
+        fluentd_conf_default = "opt/#{PACKAGE_DIR}/share/#{SERVICE_NAME}.conf.default"
         configs = [fluentd_conf]
```
Comment (kenhys) [inline fluent-package/Rakefile:482]: It seems that the .default suffix is no need to add. (as it was placed under /usr/share/)

Comment (daipom) [reply fluent-package/Rakefile:482]: I see! I will fix it!

Comment (daipom) [reply fluent-package/Rakefile:482]: Fixed!

**Comment by daipom** [comment]
Rebased to the latest master to solve conflicts

**Review by kenhys — APPROVED** [review]
LGTM.

**Comment by daipom** [comment]
Thanks for your review! Waiting CI...

**Comment by daipom** [comment]
Thanks!

**Pull request merged**